docs/insights/README.md

```diff
@@ -75,6 +75,11 @@ For example, if you want to enable "Fully Qualified ClassName In Annotation":
 You could also simplify the namespace with `use My\Insight\Namespace;`
 :::
 
+::: tip
+Although `PHPInsights` has it's own insights, it can handle Sniff from [PHP CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) and Fixer from [PHP CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer/).
```

Comment:
```suggestion
Although `PHPInsights` has it's own insights, it can handle Sniffs from [PHP CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer) and Fixers from [PHP CS Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer/).
```